spec/game_logic_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../lib/game_logic.rb'
+RSpec.describe Game do
+  describe '#display_board' do
```

Comment:
We need to call `display_board` method from our Game class to check its behavior. You'd want to check if this method returns the board and compare it to the field variable created herein [`REQUIRED`].